README.md

```diff
@@ -15,31 +15,42 @@ an AWS account.
 
 ### Install
 
-```bash
-mkvirtualenv repokid
-git clone git@github.com:Netflix/repokid.git
-cd repokid
-python setup.py develop
-repokid config config.json
-```
+There are multiple methods of install for `Repokid`:
+
+For users intending to only operate and not develop:
+
+        pip install repokid
+
+Otherwise, contributors/developers can use:
+
+        mkvirtualenv repokid
+        git clone git@github.com:Netflix/repokid.git
+        cd repokid
+        python setup.py develop
+        repokid config config.json
+
+---
 
 #### DynamoDB
 
 You will need a [DynamoDB](https://aws.amazon.com/dynamodb/) table called `repokid_roles` (specify account and endpoint in `dynamo_db` in config file).
 
 The table should have the following properties:
```

Comment:
Is there a reason we are describing the table here?  As someone new, they may get anxious that they then need to figure out how to create the table being described, but then later it seems like it will just be auto-created for them.

Is it here just as an FYI or because users need to know the structure of the DB?

Reply:
This is true, we can probably remove the description of the database structure because get_or_create will make it if it doesn't exist.

Reply:
Removal of the docs specifying a database structure makes sense once a sister PR is merged that implements the creation of a correctly specified table, naturally nit picking as this is solved by simply merging in different orders. Duly noted comments though, maybe this can be dragged into a more technically focused set of docs?

Reply:
That or just talk about it in inline code comments, an alternative I probably prefer the idea of.